test/built-ins/String/prototype/trimStart/this-value-toprimitive-meth-priority.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2017 the Valerie Young. All rights reserved.
```

Comment:
I know you're __the Valerie Young__, but maybe just your name (no _the_) is fine here. :)

Reply:
hahahaha...

Reply:
_**the Valerie Young**_